css/grid.css

```diff
@@ -1,7 +1,7 @@
 body{
 	display: grid;
-	grid-template-columns: minmax(auto, 960px) minmax(auto, 340px);
-	grid-template-rows: 25% 1fr auto;
+	grid-template-columns: minmax(auto, 75%) minmax(auto, 25%);
+	grid-template-rows: auto 3fr auto;
```

Comment:
To prevent the header from being oversized, the size was set to 25% of the total height, it is not a good idea to leave it on "auto" in this situation. Same for "3fr", restore back to both, please.

Reply:
For some height values it generatees a hiden nav bar